resources/js/app.js

```diff
@@ -4,14 +4,24 @@
  * building robust, powerful web applications using Vue and Laravel.
  */
 
+import Vue from 'vue'
+import Bugsnag from '@bugsnag/js'
+import BugsnagPluginVue from '@bugsnag/plugin-vue'
+
+Bugsnag.start({
+    apiKey: document.head.querySelector('meta[name="bugsnag-api-key"]').content,
+    plugins: [new BugsnagPluginVue()]
+})
+
 require('./bootstrap');
 var axios = require('axios');
 
 //Import SweetAlert2 for nice alert dialogs
 import Swal from 'sweetalert2';
 window.Swal = Swal;
 
-window.Vue = require('vue');
```

Comment:
Is this line replaced by the following lines?

Reply:
Line 7 AFAIK but I'm not great at JS

Reply:
I missed that line.